README.md

```diff
@@ -1,2 +1,94 @@
-# bender
-A Slack concourse CI resource that can trigger jobs
+# bender Resource
+
+A  Concourse resource that can trigger any job (deployments, releases, ...) using slack.
+Check [examples](#Examples)
+
+## Source Configuration
+
+* `slack_token`: *Required*. The slack token to be used. For more info check [slack documentation](https://api.slack.com/docs/token-types#bot).
+
+* `channel`: *Required*. The channel name to be used by the bot.
+
+* `bot_name`: *Optional*, *default `bender`*. The bot name will be used to identify and filter messages. All messages must be addressed to the bot, eg.: `@bot_name some message`.
+
+* `grammar`: *Optional.* If not defined bender will respond to all mentions `@bot_name`. If grammar is defined bender will respond to messages matching the regex expression. Use [python regular expression](https://docs.python.org/2/library/re.html) syntax. See [examples](#Examples) for inspiration.
+
+* `template`: *Optional*. A string that will be evaluated and written to `template_filename` can be used as an input file for further jobs in the pipeline.
+
+* `template_filename`: *Optional*, *default `template_file.txt`*. The file name for a generated template.
+
+## Behavior
+
+### `check`: Check for new messages that match the rules
+
+Check will report a new version;`timestamp`; if message is addressed to `@bot_name` in the selected `channel` **and** if `grammar` is defined, it *must* match the regular expression defined in `grammar`.
+
+#### `check Parameters`
+
+Check accepts no params.
+
+### `in`: Get the message
+
+A file **bender.json** will be created with the message payload.
+
+if `template` is defined it will be evaluated and written to, `template_filename`. For more info on syntax check [Template section](#Template) for example usage check the [examples](#Examples)
```

Comment:
Coma here is redundant.

Reply:
`For more info on syntax check read [Template section](#Template). For example on usage, check the [examples](#Examples)` section.`